Data-Profiles.md

```diff
@@ -41,5 +41,57 @@ Yearly 26kb / data source
 
 ![Data Source Profiles RRA Edit](images/data-source-profiles-edit2.png)
 
+### Changing data profile averages 
+
+The Cacti gui allows you to change the averages on a specific Data source profile
+some setups require a higher resolution of data and you may want to change the averaging to
+keep the most data this will come with a cost of storage due to the bigger sized RRD file
+
+To do this follow these steps
+
+#### Method 1: New system without any hosts
+
+1.) Login to cacti
```

Comment:
We do not normally include the bracket after the full stop, these should be removed.

Reply:
fixed